src/goto-checker/report_util.cpp

```diff
@@ -362,6 +367,256 @@ void output_properties_with_traces(
   }
 }
 
+void output_fault_localization_scores(
+  const fault_location_infot &fault_location,
+  messaget &log)
+{
+  log.debug() << "Fault localization scores:" << messaget::eom;
+  for(auto &score_pair : fault_location.scores)
+  {
+    log.debug() << score_pair.first->source_location
+                << "\n  score: " << score_pair.second << messaget::eom;
+  }
```

Comment:
Use `messaget::conditional_output` to avoid doing a lot of work when it most likely won't be of any value.